ACPs/TEMPLATE.md

```diff
@@ -1,13 +1,11 @@
-```text
-ACP: <PR Number>
-Title: <ACP title>
-Author(s): <a list of the author's name(s) and optionally contact info: FirstName LastName <foo@bar.com>>
-Discussions-To: <GitHub Discussion URL (POPULATED BY MAINTAINER, DO NOT SET)>
-Status: <Proposed, Implementable, Activated, Stale>
-Track: <Standards, Best Practices, Meta, Subnet>
-Replaces (*optional): <ACP number>
-Superseded-By (*optional): <ACP number>
-```
+| ACP | PR Number |
+| :--- | :--- |
+| **Title** | ACP title |
+| **Author(s)** | A list of the author's name(s) and optionally contact info: FirstName LastName ([@GitHubUsername](https://github.com/Username) or [email@address.com](mailto:email@address.com)) |
+| **Status** | Proposed, Implementable, Activated, Stale ([Discussion](https://github.com/avalanche-foundation/ACPs/discussions/XXX)) |
```

Comment:
I think retaining this will make it clearer to people just looking at the template that they shouldn't be populating it

```suggestion
| **Status** | Proposed, Implementable, Activated, Stale ([Discussion](POPULATED BY MAINTAINER, DO NOT SET) |
```